pages/reference-smart-account/guards/setModuleGuard.mdx

```diff
@@ -0,0 +1,63 @@
+import { Tabs, Callout } from 'nextra/components'
+
+# `setModuleGuard`
+
+Set a Module Guard that checks transactions initiated by the module before
+execution. 
+
+<Callout type='warning'>
+  This action can only be done via a Safe transaction.
+</Callout>
+
+<Callout type='error' emoji='🚨'>
+  Since a Module Guard has full power to block Safe transaction execution
+  initiated via a module, a broken Module Guard can cause a denial of service
+  for the Safe Modules. Make sure to carefully audit the Module Guard code and
+  design recovery mechanisms.
+</Callout>
+
+## Usage
+
+{/* <!-- vale off --> */}
+
+<Tabs items={['example.sol']}>
+  <Tabs.Tab>
+    ```solidity
+    interface ISafe {
+        function setModuleGuard(address moduleGuard) external;
+    }
+
+    contract Example {
+        function example() ... {
+            (ISafe safe).setModuleGuard("0x...");
+        }
+    }
+    ```
+  </Tabs.Tab>
+</Tabs>
+
+{/* <!-- vale on --> */}
+
+## Parameters
+
+### `moduleGuard`
+
+- **Type:** `address`
+
+The address of the Module Guard to be used or the zero address to disable the Module Guard.
```

Comment:
This is differently written in another file (https://github.com/safe-global/safe-docs/pull/624/files#diff-2ac0e1982858d377d8e40cce1914fa333f03795a62fef38d0abe2b0e0a1f1b4dR40). Can you please make it consistent? Preference for written numbers.